RealEstate/apps/core/templates/core/realtorHome.html

```diff
@@ -39,25 +15,27 @@
 				<li id="row_{{couple.id}}" class="list-group-item couple_row">
 			{% endif %}
 			
-					<div class="row">
-						<div class="col-sm-7 vcenter couple-container">
-							{% for homebuyer in homebuyers %}
-								<h4>
-									{% if isPending %}
-										{{homebuyer.email}}
-										{% if not homebuyer.registered %}<span class="red-alert">*</span> {% endif %}
-										<a href="mailto:{{homebuyer.email}}" target="_top" class="mail"><span id="mail_p{{couple.id}}" class="glyphicon glyphicon-envelope vcenter mail-icon" ></span></a>
-									{% else %}
-										{{homebuyer}}<a href="mailto:{{homebuyer.email}}" target="_top" class="mail"><span id="mail_{{couple.id}}" class="glyphicon glyphicon-envelope vcenter mail-icon" ></span></a>
-									{% endif %}
-								</h4>
-							{% endfor %}
-						</div>
-						<div class="col-sm-5 vcenter report-container">
-							<center>
-								<a class="report" href="{{couple.report_url}}">Report</a>
-							</center>
-						</div>
+					<div class="row home_row" id="row_{{couple.id}}">
+                        <div class="row-con">
+        					<div class="col-sm-6">
+        						{% for homebuyer in homebuyers %}
+        							<h4>
+        								{% if isPending %}
+        									{{homebuyer.email}}
+        									{% if not homebuyer.registered %}<span class="red-alert">*</span> {% endif %}
+        									<a href="mailto:{{homebuyer.email}}" target="_top" class="mail"><span id="mail_p{{couple.id}}" class="glyphicon glyphicon-envelope vcenter mail-icon" ></span></a>
+        								{% else %}
+        									{{homebuyer}}<a href="mailto:{{homebuyer.email}}" target="_top" class="mail"><span id="mail_{{couple.id}}" class="glyphicon glyphicon-envelope vcenter mail-icon" ></span></a>
+        								{% endif %}
+        							</h4>
+        						{% endfor %}
+        					</div>
+                            <div class="hbdash-center">
+        						<center>
+        							<a class="report" href="{{couple.report_url}}">Report</a>
+        						</center>
+                            </div>        					
+                        </div>
```

Comment:
Something weird happened to this row. I see a column of 6. But I'm not sure where the right column went to. This may be intentional, but I just wanted you to be aware of it.